activemodel/test/cases/type/float_test.rb

```diff
@@ -21,9 +21,10 @@ def test_type_cast_float_from_invalid_string
       def test_changing_float
         type = Type::Float.new
 
-        assert type.changed?(5.0, 5.0, "5wibble")
```

Comment:
Can we add a test to make sure `changed?` returns true?

Reply:
I think `changed?` should return false for this case.

This test succeeds for current master:
```
  def test_float_change
    price = Price.create!(float_col: "5wibble")
    assert_equal 5.0, price.float_col
  end
```

Am I missing something?

Reply:
I see. I'll update PR tomorrow.